Source/AwsCommonRuntimeKit/io/TlsConnectionOptions.swift

```diff
@@ -3,37 +3,37 @@
 
 import AwsCIo
 
-public final class TlsConnectionOptions {
+public struct TlsConnectionOptions: CStruct {
 	private let allocator: Allocator
-    var rawValue: UnsafeMutablePointer<aws_tls_connection_options>
+	public var context: TlsContext
+	public var alpnList: [String]
```

Comment:
trivial: have it be optional `alpnList: [String]?`

ALPN is an optional extension during TLS handshakes. Probably in our code passing an empty array, resulting in an empty string, results in the extension not being used ... but I'd rather our API be more explicit about it

Reply:
Thanks, fixed.